src/Tasks/Common/Resources/Strings.resx

```diff
@@ -692,7 +692,7 @@ The following are names of parameters or literal values and should not be transl
   </data>
   <data name="WorkloadNotInstalled" xml:space="preserve">
     <value>NETSDK1147: To build this project, the following workloads must be installed: {0}
-To install these workloads, run the following command: {1}</value>
+To install these workloads, run the following command: dotnet workload restore</value>
```

Comment:
Is there any context we need to give to the Loc team that `dotnet workload restore` is a non-localizable value? Or do they have that context already from prior experience?

Reply:
That's a good question. I'm not sure if there is a way to mark these that way and I've definitely seen items like this get localized in the past. @cristianosuzuki77 what can we do to mark this so the localizers don't translate the new text "dotnet workload restore" as those are CLI commands?

Reply:
I didn't find any examples in the sdk repo but I did find some in other repos like msbuild and templating where messages were included in the comments to the loc team. As such, I've added a comment here about that. If we don't hear back from Cristiano, we can proceed with this and see if we get pushback/feedback on this approach.